example/contact-bot.ts

```diff
@@ -78,6 +78,20 @@ async function main() {
   log.info('Bot', '#######################')
   log.info('Bot', 'Contact number: %d\n', contactList.length)
 
+  /**
+   * List Brand Contact
+   */
+  for (let i = 0; i < contactList.length; i++) {
+    const contact = contactList[i]
+    if (contact.brand() === null) {
```

Comment:
The `contact` here should already `ready()`-ed, because Wechaty should make sure it is.

If it is not, then Wechaty must have a bug in somewhere...